Description: This is an update that fixed hinting problem of
 Liberation Sans Bold character "u" at particular point size.
Origin: https://admin.fedoraproject.org/updates/FEDORA-2012-1644/liberation-fonts-1.07.2-2.fc16
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=616620
Bug-Redhat: https://bugzilla.redhat.com/show_bug.cgi?id=715309

--- trunk.orig/src/LiberationSans-Bold.sfd
+++ trunk/src/LiberationSans-Bold.sfd
@@ -3103,7 +3103,7 @@ NameList: Adobe Glyph List
 DisplaySize: -36
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 1113420 44 17
+WinInfo: 1113400 38 14
 BeginPrivate: 0
 EndPrivate
 BeginChars: 1114132 681
@@ -18567,118 +18567,133 @@ StartChar: u
 Encoding: 117 117 88
 Width: 1251
 Flags: W
-HStem: -20 210<479.297 667.902> 0 21G<847.5 1116> 1062 20G<127 408 827 1108>
-VStem: 127 281<269.177 1082> 827 281<3.2583 215 304.984 1082> 844.5 271.5<0 96.0013>
 TtInstrs:
-SVTCA[y-axis]
-PUSHB_2
+NPUSHB
+ 46
+ 31
+ 16
+ 9
+ 10
+ 0
+ 76
+ 20
+ 11
+ 70
+ 223
+ 14
+ 1
+ 14
+ 64
+ 36
+ 40
+ 72
+ 14
+ 37
+ 144
+ 37
+ 160
+ 37
+ 2
+ 112
+ 37
+ 144
+ 37
+ 160
+ 37
+ 176
+ 37
+ 240
+ 37
+ 5
+ 239
+ 37
+ 1
+ 1
+ 70
+ 239
+ 34
+ 255
+ 34
+ 2
+ 34
+PUSHW_1
+ -64
+NPUSHB
+ 14
+ 9
+ 12
+ 72
+ 34
+ 25
+ 6
+ 80
+ 29
+ 22
+ 20
  21
- 8
+ 12
+ 0
+ 15
+SVTCA[y-axis]
 MIAP[rnd]
-PUSHB_2
- 2
- 8
+SHP[rp2]
 MIAP[rnd]
-PUSHB_1
- 11
-MDRP[min,rnd,black]
-PUSHB_2
- 7
- 15
 MIAP[rnd]
-PUSHB_1
- 16
+MIRP[min,rnd,black]
 SHP[rp1]
 SVTCA[x-axis]
-PUSHB_1
- 28
 MDAP[rnd]
-PUSHB_1
- 6
-MDRP[rp0,rnd,white]
-PUSHB_1
- 9
-MDRP[min,rnd,black]
-PUSHB_1
- 9
+CALL
+DELTAP1
+MIRP[min,rnd,black]
+DELTAP1
+DELTAP2
+DELTAP3
 SRP0
-PUSHB_1
- 15
-MDRP[rp0,min,rnd,grey]
-PUSHB_2
- 24
- 27
-SHP[rp2]
+MDRP[rp0,min,rnd,white]
+CALL
+DELTAP1
+MIRP[min,rnd,black]
 SHP[rp2]
-PUSHB_1
- 18
-MDRP[min,rnd,black]
-PUSHB_1
- 18
-SRP0
-PUSHB_1
- 23
-MDRP[min,rnd,black]
-PUSHB_1
- 23
-MDAP[rnd]
-PUSHB_1
- 18
-SRP0
-PUSHB_1
- 29
-MDRP[rp0,rnd,white]
-PUSHB_2
- 23
- 15
-SRP1
-SRP2
-PUSHB_1
- 0
-IP
-SVTCA[y-axis]
-PUSHB_2
- 11
- 21
-SRP1
-SRP2
-PUSHB_1
- 23
-IP
-PUSHB_1
- 7
-SRP1
-PUSHB_2
- 0
- 27
-IP
-IP
-IUP[y]
 IUP[x]
+IUP[y]
+SVTCA[y-axis]
+CALL
 EndTTInstrs
 LayerCount: 2
 Fore
 SplineSet
-831 215 m 1,0,1
- 724.008 -20 724.008 -20 483 -20 c 0,2,3
- 299.009 -20 299.009 -20 207.5 99.5 c 0,4,5
- 127 204.624 127 204.624 127 395 c 2,6,-1
- 127 1082 l 1,7,-1
- 408 1082 l 1,8,-1
- 408 475 l 2,9,10
- 408 190 408 190 600 190 c 0,11,12
- 701.47 190 701.47 190 764 276.5 c 0,13,14
- 827 363.65 827 363.65 827 502 c 2,15,-1
- 827 1082 l 1,16,-1
- 1108 1082 l 1,17,-1
- 1108 242 l 2,18,19
- 1108 104 1108 104 1116 0 c 1,20,-1
- 848 0 l 1,21,22
- 847 12 847 12 844.5 38.5 c 128,-1,23
- 842 65 842 65 840.5 97 c 2,24,-1
- 837.5 161 l 2,25,26
- 836 193 836 193 836 215 c 1,27,-1
- 831 215 l 1,0,1
+408 1082 m 1,0,-1
+ 408 475 l 2,1,2
+ 408 412 408 412 418 359.5 c 128,-1,3
+ 428 307 428 307 451 269.5 c 128,-1,4
+ 474 232 474 232 510.5 211 c 128,-1,5
+ 547 190 547 190 600 190 c 0,6,7
+ 651 190 651 190 692.5 212.5 c 128,-1,8
+ 734 235 734 235 764 276.5 c 128,-1,9
+ 794 318 794 318 810.5 375.5 c 128,-1,10
+ 827 433 827 433 827 502 c 2,11,-1
+ 827 1082 l 1,12,-1
+ 1108 1082 l 1,13,-1
+ 1108 242 l 2,14,15
+ 1108 208 1108 208 1108.5 170.5 c 128,-1,16
+ 1109 133 1109 133 1110.5 99.5 c 128,-1,17
+ 1112 66 1112 66 1113.5 39.5 c 128,-1,18
+ 1115 13 1115 13 1116 0 c 1,19,-1
+ 848 0 l 1,20,21
+ 847 12 847 12 844.5 38.5 c 128,-1,22
+ 842 65 842 65 840.5 97 c 128,-1,23
+ 839 129 839 129 837.5 161 c 128,-1,24
+ 836 193 836 193 836 215 c 1,25,-1
+ 831 215 l 1,26,27
+ 775 92 775 92 688.5 36 c 128,-1,28
+ 602 -20 602 -20 483 -20 c 0,29,30
+ 386 -20 386 -20 318 12 c 128,-1,31
+ 250 44 250 44 207.5 99.5 c 128,-1,32
+ 165 155 165 155 146 231 c 128,-1,33
+ 127 307 127 307 127 395 c 2,34,-1
+ 127 1082 l 1,35,-1
+ 408 1082 l 1,0,-1
 EndSplineSet
 EndChar
 
